.codecov.yml

```diff
@@ -0,0 +1,20 @@
+#
```

Comment:
note the file move. The github action seems to look for this file in the root rather than in `.github`